cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/Language.kt

```diff
@@ -75,11 +75,12 @@ data class ImplicitCast(override var depthDistance: Int) : CastResult(depthDista
  * the [Node.language] property.
  */
 abstract class Language<T : LanguageFrontend<*, *>> : Node() {
+
     /** The file extensions without the dot */
     abstract val fileExtensions: List<String>
 
     /** The namespace delimiter used by the language. Often, this is "." */
-    abstract val namespaceDelimiter: String
```

Comment:
Yeah, this was somehow necessary because otherwise the namespace delimiter is null in the constructor of language, which then crashes. It is a bit weird that this worked in the past but now it crashed :( I think having `.` as the default is fairly reasonable.